.pipelines/azure-pipelines/azure-pipelines-REFramework.yml

```diff
@@ -0,0 +1,36 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - develop
+    - master/REFramework/*
+  paths:
+    include:
+    - REFramework/*
+    - .pipelines/*
+
+
+pr: none
+  # branches:
+  #   include:
+  #   - develop
+  #   - master/REFramework/*
+  # paths:
+  #   include:
+  #   - REFramework/*
+  #   - .pipelines/*
+
+
+variables:
+  - template: .pipelines/variables/common.yml
+  - name: projectPath
+    value: 'REFramework'
+  - name: projectNuspec
+    value: 'UiPath.Template.REFramework.nuspec'
+
+jobs:
+  - template: .pipelines/jobs/ci.yml
```

Comment:
nok, location is relative to your current location

Reply:
fixed